<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tutoría UCO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_main">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      
      <widget class="QWidget" name="pageLogin">
       <layout class="QVBoxLayout" name="verticalLayout_login">
        <item>
         <spacer name="verticalSpacer_1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelTitle">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Acceso UCO Tutorías</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputUser">
          <property name="placeholderText">
           <string>Usuario (ej: alumno1)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputPass">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Contraseña</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLogin">
          <property name="text">
           <string>Entrar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>

      <widget class="QWidget" name="pageChat">
       <layout class="QVBoxLayout" name="verticalLayout_chat">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_header">
          <item>
           <widget class="QLabel" name="labelInfo">
            <property name="text">
             <string>Bienvenido</string>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnLogout">
            <property name="text">
             <string>Cerrar Sesión</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="chatDisplay">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_input">
          <item>
           <widget class="QLineEdit" name="inputMsg">
            <property name="placeholderText">
             <string>Escribe tu mensaje...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEnviar">
            <property name="text">
             <string>Enviar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_actions">
          <item>
           <widget class="QPushButton" name="btnAsignar">
            <property name="text">
             <string>Asignar Tutor Automático</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnBorrarChat">
            <property name="text">
             <string>Eliminar Chat Actual</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>